src/main/java/com/google/api/codegen/configgen/transformer/LanguageTransformer.java

```diff
@@ -161,4 +161,21 @@ public String rewrite(String input) {
       return input.replaceAll(pattern, replacement);
     }
   }
+
+  private static class PythonLanguageFormatter implements LanguageFormatter {
+    private static final RewriteRule PYTHON_REWRITE_RULE =
```

Comment:
This duplicates the `commonRewriteRules`. How about passing it in as a constructor parameter?

Reply:
Done.